Kubernetes/containerd/start.ps1

```diff
@@ -0,0 +1,493 @@
+
+[CmdletBinding(PositionalBinding=$false)]
+Param
+(
+    [parameter(ParameterSetName='Default', Mandatory = $true, HelpMessage='Kubernetes Config File')] [string]$ConfigFile,
+    [parameter(ParameterSetName='Default', Mandatory = $false, HelpMessage='Kubernetes cluster cidr')] [string]$ClusterCIDR,
+    [parameter(ParameterSetName='Default', Mandatory = $false, HelpMessage='Kubernetes pod service cidr')] [string]$ServiceCIDR,
+    [parameter(ParameterSetName='Default', Mandatory = $false, HelpMessage='Kubernetes DNS Ip')] [string]$KubeDnsServiceIP,
+    [parameter(ParameterSetName='Default', Mandatory = $false, HelpMessage='Skip downloading binaries')] [switch] $SkipInstall,
+
+    [parameter(ParameterSetName='OnlyInstall', Mandatory = $false)] [switch] $OnlyInstall,
+
+    [parameter(ParameterSetName='StopNode', Mandatory = $false)] [switch] $StopNode
+)
+$ProgressPreference = 'SilentlyContinue'
+
+$kubernetesPath = "C:\k"
+$cniDir = Join-Path $kubernetesPath cni
+$cniConfigDir = Join-Path $cniDir config
+$containerdPath = "$Env:ProgramFiles\containerd"
+$flanneldPath = "C:\flannel"
+$flanneldConfPath = "C:\etc\kube-flannel"
+$lcowPath = "$Env:ProgramFiles\Linux Containers"
+
+$networkMode = "L2Bridge"
+$networkName = "cbr0"
+$kubeDnsSuffix="svc.cluster.local"
+$kubeletConfigPath = Join-Path $kubernetesPath "kubelet-config.yaml"
+$cniConfig = Join-Path $cniConfigDir "cni.conf"
+
+# create all the necessary directories if they don't already exist
+New-Item -ItemType Directory -Path $kubernetesPath -Force > $null
+New-Item -ItemType Directory -Path $cniConfigDir -Force > $null
+New-Item -ItemType Directory -Path $containerdPath -Force > $null
+New-Item -ItemType Directory -Path $flanneldPath -Force > $null
+New-Item -ItemType Directory -Path $flanneldConfPath -Force > $null
+New-Item -ItemType Directory -Path $lcowPath -Force > $null
+
+# Setup functions
+
+Function CreateVMSwitch() {

```

Comment:
Why do we have to create a VMSwitch?

Reply:
This was just functionality I found would be useful to provide to the user in my scripts, if it's not expected in SDN scripts I'll remove it